scripts/jenkins-release.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+[ -d venv ] || virtualenv venv
+pip install -e .
+
+if [ "$ENVIRONMENT" = "Select one" -o "$ENVIRONMENT" = "" ]; then
```

Comment:
I went with environment because it's a more commonly understood term. It's the language that's used throughout GDS and the service manual.